bucoffea/vbfhinv/definitions.py

```diff
@@ -150,10 +104,60 @@ def vbfhinv_accumulator(cfg):
     items['photon_eta0'] = Hist("Counts", dataset_ax, region_ax, eta_ax)
     items['photon_phi0'] = Hist("Counts", dataset_ax, region_ax, phi_ax)
 
-    items['photon_pt0_recoil'] = Hist("Counts", dataset_ax, region_ax, pt_ax, recoil_ax)
+    # Create histograms for each JES/JER variation
+    for var in variations: 
```

Comment:
Lets only make plots we are really interested for all the variations.

I.e. to first order just mjj and maybe jet and met pt.

Also, instead of defining N histograms, why dont we define e.g. a `recoil_variation` histogram that in addition to all the axis the normal `recoil` histogram has, also has a `variation` axis, so that it can be filled as:

```
items["recoil"].Fill(recoil=recoil_jesup, variation="jesup",...)
```

?

Reply:
You're right, I think I can remove histograms that are less interesting for now and maybe go with just a few histograms as you said.

Introducing a new axis is actually a great idea, thank you. I can work on this so we don't have a separate histogram for each variation.